modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -318,6 +318,21 @@ export default {
       this.uploading = false;
       await this.getMedia();
 
+      if (Array.isArray(imgIds) && imgIds.length && this.items.length === 0) {
```

Comment:
"We uploaded and nothing got selected..." Yes I guess that works. The behavior would be different if the file wasn't of the right type etc. Here the upload succeeds but the selection doesn't take it. 👍